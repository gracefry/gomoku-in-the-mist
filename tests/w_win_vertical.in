place A1
place B1
place A3
place C1
place A5
place D1
place A7
place E1
place A9
place F1