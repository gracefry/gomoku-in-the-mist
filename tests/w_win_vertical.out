White wins!
A1B1A3C1A5D1A7E1A9F1
Thank you for playing!
